Added some gui improvements

FelixBrendel opened the pull request:
Added:
- window title
- command line argument parser
   - `-nosgf` this flag will prevent the sgf output
   - `-hideoutput` this will hide the outputbox at the bottom
   - more flags can be added easily
- added stone shadows
- the last played position is now marked with a dot
- used Sabakis background image
![image](https://user-images.githubusercontent.com/5005975/33855498-5964299a-dec5-11e7-86d4-d1327eeee8c1.png)

FelixBrendel committed:
added lastPlayed marker and options for nosgf and no textbox

FelixBrendel committed:
cleanup

FelixBrendel committed:
added Sabakis background image

FelixBrendel committed:
added stone shadows

FelixBrendel committed:
reduced shadow amount

FelixBrendel committed:
fixed shadow rendering, alos pramametrized the gimmicks

fsparv commented:
This image looks great! I love it. Can you link me the licensing for the image (hopefully documented by the place you adopted it from as something Apache 2.0 compatible). I'll try to look at the code in this soon, but I've had a major issue in my personal life... Town sewer backed up into my basement. So it could be a few days... 

Quick peek shows you created your own build script. Please remove that. Gradle is the build tool for this project, and I develop on linux so these sort of things should be added as custom targets written in groovy code in the gradle build, not .bat files. That keeps the project usable by folks on multiple platforms. Alternately just remove it and just keep it local for yourself.

As for command line ops: I generally use this library to keep things consistent and avoid the myriad of option parsing pitfalls: https://github.com/docopt/docopt.java

fsparv commented on src/main/java/leelawatcher/gui/LeelaWatcher.java:
```diff
@@ -113,8 +131,18 @@ public void windowClosing(WindowEvent e) {
     splitPane.setRightComponent(textScrollPane);
-    leelaOutputTextArea = new JTextArea();
-    leelaOutputTextArea.setRows(8);
+
```
You didn't read the comment at the top of this method did you :) ? This method must not be edited. it is generated by a tool in my IDE. This code needs to move to createUIComponents() or main() before the call to frame.pack() ... Also, pls leave the rows > 4 for the text area so one has a chance to see the game ending message.

fsparv commented:
Whoops, also looks like you checked in some network weight files :) need to avoid that pls.

FelixBrendel committed:
removed buildscript and weights file

FelixBrendel committed:
fixed the untouchable gui method

FelixBrendel commented:
- The background image was taken from Sabaki (https://github.com/yishn/Sabaki) I am not a license expert but it says MIT License which allows Commercial use, Modification, Distribution and Private use
- I added my build script to the gitignore file so it is not in the repository anymore
- also removed weight files
- fixed the method that should not be touched, set the outbut box rows to 4 again

The only thing I did not do is the command line argument parser, because I do not know how to include it, or set a dependency. I am not a Java dev. Could you help me with this one?

fsparv commented:
ok sure, I'll help with the command line lib, perhaps later this week. Thx for your effort so far!

billyswong commented:
If we are replacing background image, any chance for embedding tiny coordinate letters too?

FelixBrendel commented:
The background image is not square. But instead it is tiled. So printing the letters on the image does not work. However coordinates are not impossible and can be done after that :)

fsparv commented:
So I have this merged in locally, but I've realized we have a problem and I've run out of time to work on it today. If I accept this, the huge weight files will still be in the history (large files increase the download size of the repo forever), also you have a commit where you seem to have re-added everything with identical contents which is pretty confusing. I need you to re-do your commits on a new branch and submit a new clean pull request. You should be able to do this by creating a branch and then cherry picking your revisions and before committing removing the undesired files. I don't know if you are in the habit of reading your diffs before check-in (maybe you just forgot) but it's a good habit to avoid snafu's like this.

fsparv commented:
Any chance this can get cleaned up soon? I'm enjoying the features locally but I'd like to have a set of commits I can push to the rest of the world without permanently burdening my public repo with weight files and some sort of strange small change to all the image files.

FelixBrendel commented:
Sorry I have been busy with university stuff and Christmas stress.. Will try to do it tomorrow as a Christmas present ^^
